backend/src/main/java/gov/cdc/usds/simplereport/config/CorsProperties.java

```diff
@@ -2,10 +2,8 @@
 
 import java.util.List;
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.ConstructorBinding;
 
 @ConfigurationProperties(prefix = "simple-report.cors")
-@ConstructorBinding
```

Comment:
Why is the @ConstructorBinding removed completely? Because the class has only one constructor and it falls as a default? 🤔

Reply:
[With the upgrade](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#constructingbinding-no-longer-needed-at-the-type-level), it's no longer needed with `@ConfigurationProperties` on the type level and only needed when there are multiple constructors.